### ISSUE TITLE
Proxy is not defined in Node 4

### ISSUE THREAD

**Issue by nlindley** [atom/fs-plus#38]
The new export using `Proxy` in 2.10.0 breaks in Node 4 since `Proxy` is not available. I would consider this a breaking change, but at the very least, please specify engines in the package.json.

**Comment by I-n-d-e-r** [comment]
Confirm. Previous release works fine.

Node:	v6.9.5
Npm:	v3.10.10

```js
/builddir/build/BUILD/atom-1.14.4/apm/node_modules/atom-package-manager/node_modules/fs-plus/lib/fs-plus.js:708
  module.exports = new Proxy({}, {
                       ^
ReferenceError: Proxy is not defined
```js

**Comment by ashleygwilliams** [comment]
i also have this issue.

**Comment by ashleygwilliams** [comment]
related: https://github.com/atom/fs-plus/pull/39

**Comment by ashleygwilliams** [comment]
ps if you ran into this problem because you were using highlights- i've fixed it over there for the moment.

**Comment by ashleygwilliams** [comment]
https://github.com/atom/highlights/pull/51

**Comment by as-cii** [comment]
Apologies for this issue, I should have been more careful when bumping the minor version. I will publish a new patch version that removes the `Proxy` object tomorrow morning.

**Comment by as-cii** [comment]
I have just released fs-plus 2.10.1 and it should now work again on Node 4. Thanks everyone for the feedback and sorry again for introducing this problem!

**Comment by crapthings** [comment]
Proxy is not defined

node 8.9.2 ??

should i downgrade node or fs-plus ?
